esrally/driver/runner.py

```diff
@@ -2892,6 +2892,7 @@ async def __call__(self, es, params):
         query = mandatory(params, "query", self)
         body = params.get("body", {})
         body["query"] = query
+        body["version"] = params.get("version", "2024.04.01.🚀")
```

Comment:
:rocket:

Reply:
Just curious, what's the purpose of UTF emoji? To have an option to release 2 versions in the same day? Or have an easy-to-remember mnemonic?

Reply:
@pquentin was kind enough and provided context behind the emoji. As this is just for fun and optional, I would actually prefer to skip it to avoid questions such as mine in the future from Rally code users.

Reply:
Dropped.